packages/cbioportal-frontend-commons/src/api/remoteData.ts

```diff
@@ -47,3 +47,17 @@ export const remoteData: MobxPromiseFactory = function<R>(
     });
     return mobxPromise;
 };
+
+export function mobxPromiseResolve<R>(value: R): MobxPromise<R> {
```

Comment:
I made this in the process but didn't end up using it. Seems like it could end up being useful anyway though. Just LMK if you think I should delete it @alisman

Reply:
not sure i understand how it would be used. example?

Reply:
if you had a component that accepted a mobxpromise as a prop, but in your case the result was not asynchronously computed, then instead of refactoring the component or creating a remoteData you could just wrap the result in this to meet the interface